examples/pytorchjob-example.yaml

```diff
@@ -0,0 +1,93 @@
+apiVersion: "kubeflow.org/v1alpha1"
+kind: StudyJob
+metadata:
+  namespace: kubeflow
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: pytorchjob-example
+spec:
+  studyName: pytorchjob-example
+  owner: crd
+  optimizationtype: maximize
+  objectivevaluename: accuracy
+  optimizationgoal: 0.99
+  requestcount: 4
+  metricsnames:
+    - accuracy
+  parameterconfigs:
+    - name: --lr
+      parametertype: double
+      feasible:
+        min: "0.01"
+        max: "0.05"
+    - name: --momentum
+      parametertype: double
+      feasible:
+        min: "0.5"
+        max: "0.9"
+  workerSpec:
+    retain: true
+    goTemplate:
+        rawTemplate: |-
+          apiVersion: "kubeflow.org/v1beta1"
+          kind: PyTorchJob
+          metadata:
+            name: {{.WorkerID}}
+            namespace: kubeflow
+          spec:
+           pytorchReplicaSpecs:
+            Master:
+              replicas: 1
+              restartPolicy: Never
+              template:
+                spec:
+                  containers:
+                    - name: pytorch
+                      image: johnugeorge/pytorch-mnist-with-summary:0.4
```

Comment:
Can you check in the source code for this? Also this should be on gcr.io.

Reply:
Can you update the image?

Reply:
@johnugeorge Done. Pushed as gcr.io/kubeflow-ci/pytorch-mnist-with-summary:0.4.

Reply:
updated